### PR TITLE
ahrocksdb: ignore CI failures

### PR DESCRIPTION
**Pull request by emillon**
The error message is:

> installed RocksDB installation is too old: found 6.11, expected 5.14 minimum

Which is actually fine! There's a logic error which has been fixed upstream in ahrefs/ocaml-ahrocksdb@d74dc6f8337557c23b66c2a2488957d7ce3c278c.

This PR ignores CI errors caused by this.

### CODE REVIEW

File: packages/ahrocksdb/ahrocksdb.0.2.0/opam [hunk 1]
```diff
@@ -42,6 +42,13 @@ build: [
   ["dune" "runtest"] {with-test}
```
Comment (mseri) [inline packages/ahrocksdb/ahrocksdb.0.2.0/opam:42]:
```suggestion
  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
```

File: packages/ahrocksdb/ahrocksdb.0.2.1/opam [hunk 2]
```diff
@@ -40,6 +40,13 @@ build: [
   ["dune" "runtest"] {with-test}
```
Comment (mseri) [inline packages/ahrocksdb/ahrocksdb.0.2.1/opam:40]:
```suggestion
  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
```

File: packages/ahrocksdb/ahrocksdb.0.2.0/opam [hunk 3]
```diff
@@ -39,9 +39,16 @@ depends: [
 build: [
   ["dune" "subst"] {dev}
   ["dune" "build" "-p" name "-j" jobs]
-  ["dune" "runtest"] {with-test}
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
 ]
 x-ci-accept-failures: ["debian-unstable"]
```
Comment (kit-ty-kate) [inline packages/ahrocksdb/ahrocksdb.0.2.0/opam:44]: I'm not sure it's worth patching packages especially if the latest version fixed the issue already. I'd recommend just ignoring the failure instead
```suggestion
x-ci-accept-failures: ["debian-11" "debian-unstable"]
```

**Comment by emillon** [comment]
@kit-ty-kate I switched this to just ignore CI errors.

**Comment by mseri** [comment]
Since the fix was already there I don’t see why not keeping it. Anyway I am fine to merge it as is or with the fix

**Comment by kit-ty-kate** [comment]
Thanks. Sorry for the wait

**Pull request merged**